server/frontend/package.json

```diff
@@ -1,86 +1,58 @@
-    "scripts": {
-        "codecov": "codecov -X gcov -p ../..",
-        "test": "jest --verbose --coverage",
-        "build": "webpack --config webpack.dev.js",
-        "start": "webpack --config webpack.dev.js --watch",
-        "production": "webpack --config webpack.prod.js",
-        "lint": "eslint ./"
-    },
-    "dependencies": {
-        "axios": "^0.28.1",
-        "d3": "^7.8.2",
-        "handlebars": "^4.7.7",
-        "js-base64": "^3.6.1",
-        "lodash": "^4.17.21",
-        "sweetalert": "^2.1.2",
-        "vue": "^2.7.15",
-        "vue-popperjs": "^2.3.0",
-        "vue-router": "^3.5.1",
-        "vue-spinner": "^1.0.4"
-    },
-    "devDependencies": {
-        "@babel/core": "^7.15.0",
-        "@babel/preset-env": "^7.15.0",
-        "@eslint/compat": "^1.1.0",
-        "@eslint/eslintrc": "^3.1.0",
-        "@eslint/js": "^9.6.0",
-        "@testing-library/dom": "^8.1.0",
-        "@testing-library/vue": "^5.8.2",
-        "@vue/test-utils": "^1.2.2",
-        "babel-core": "^7.0.0-bridge.0",
-        "babel-jest": "^27.0.6",
-        "codecov": "^3.8.3",
-        "css-loader": "^5.2.4",
-        "eslint": "^8.57.0",
-        "eslint-config-prettier": "^9.1.0",
-        "eslint-plugin-prettier": "^5.1.3",
-        "eslint-plugin-vue": "^9.26.0",
-        "globals": "^15.7.0",
-        "jest": "^27.0.6",
-        "mini-css-extract-plugin": "^1.6.0",
-        "postcss": "^8.3.5",
-        "prettier": "^3.3.2",
-        "vue-jest": "^3.0.7",
-        "vue-loader": "^15.9.6",
-        "vue-template-compiler": "^2.7.15",
-        "webpack": "^5.89.0",
-        "webpack-cli": "^5.1.4",
-        "webpack-merge": "^5.10.0"
-    },
-    "jest": {
-        "moduleFileExtensions": [
-            "js",
-            "json",
-            "vue"
-        ],
-        "collectCoverageFrom": [
-            "**/*.{js,jsx}",
-            "**/*.vue",
-            "!**/tests/**",
-            "!**/coverage/**",
-            "!**/dist/**",
-            "!**/webpack.*.js"
-        ],
-        "moduleNameMapper": {
-            "d3": "<rootDir>/node_modules/d3/dist/d3.min.js"
-        },
-        "transformIgnorePatterns": [
-            "/node_modules/(?!vue-spinner)"
-        ],
-        "transform": {
-            ".*\\.(vue)$": "vue-jest",
-            ".*\\.(js)$": "babel-jest"
-        },
-        "testEnvironment": "jsdom"
-    },
-    "babel": {
-        "presets": [
-            "@babel/preset-env"
-        ]
-    }
+  "name": "fuzzmanager",
+  "version": "0.1.0",
+  "description": "FuzzManager frontend project",
+  "main": "index.js",
+  "scripts": {
+    "codecov": "codecov -X gcov -p ../..",
+    "test": "jest --verbose --coverage",
+    "build": "webpack --config webpack.dev.js",
+    "start": "webpack --config webpack.dev.js --watch",
+    "production": "webpack --config webpack.prod.js",
+    "lint": "eslint ./",
+    "format": "prettier --write ./"
+  },
+  "dependencies": {
+    "@vue/compat": "^3.1.0",
```

Comment:
I understood that `@vue/compat` was to be removed before this would be submitted. Was this missed?